management/src/test/java/org/ehcache/docs/ManagementTest.java

```diff
@@ -99,8 +99,10 @@ public void usingManagementRegistry() throws Exception {
 
         // hit count is a sampled stat, for example its values could be [0,0,3,4].
         // In the present case, only the last value is important to us , the cache was eventually hit 4 times
-        int mostRecentIndex = onHeapStore_Hit_Count.getValue().length - 1;
-        onHeapHitCount = onHeapStore_Hit_Count.getValue()[mostRecentIndex].getValue();
+        if (onHeapStore_Hit_Count.getValue().length > 0) {
```

Comment:
Well, might be one - but then a similar fix is missing for `ClusteringManagementServiceTest` - see https://ehcache.ci.cloudbees.com/job/ehcache3-pull-requests/lastCompletedBuild/jdk=Oracle%20JDK%201.8(latest),label_exp=standard/testReport/org.ehcache.clustered.management/ClusteringManagementServiceTest/test_stats_collection/